packages/next/server/config-shared.ts

```diff
@@ -355,11 +355,16 @@ export interface NextConfig extends Record<string, any> {
   /**
    * By default, Next.js will automatically inline font CSS at build time
    *
-   * @default true
+   * @default { inlineFonts: true }
    * @since version 10.2
    * @see [Font Optimization](https://nextjs.org/docs/basic-features/font-optimization)
    */
-  optimizeFonts?: boolean
+  optimizeFonts?:
+    | boolean
+    | {
+        inlineFonts: boolean
+        experimentalAdjustFallbacks?: boolean
```

Comment:
As discussed in Slack, I think we should break this config out into a separate config under the `experimental` group so we aren't mixing stable and unstable config values similar to how we separate the stable and unstable image configs.

e.g. this could remain `optimizeFonts?: boolean` and then a separate `experimental.adjustFontFallbacks?: boolean`

Reply:
Ah! I understand what you mean now. We've discussed this quite a bit about the flag setup in design and RFC phase before deciding on this structure.

Do you think it would make sense to keep `optimizeFonts: { inlineFonts: boolean }` as an object? 
and add `experimental.adjustFontFallbacks?: boolean` 

The main reason is we want to convey that `optimizeFonts` will also change?

What would be the best structure for a minor release?

Reply:
I've made this change. Please review